spec/yabeda/dsl/class_methods_spec.rb

```diff
@@ -106,4 +106,16 @@
       it("defines method on root object") { is_expected.to be_a(Yabeda::Histogram) }
     end
   end
+
+  describe ".general_tag" do
+    subject { Yabeda.general_tags }
+
```

Comment:
Let's add assertion for default behavior here:
```suggestion

it { is_expected.to eq({}) }

```

Reply:
I add it to `spec/yabeda_spec.rb`. Should I move it to this file and keep it in `yabeda_spec`?

Reply:
Ah, now I see. No, then it is not needed anymore